packages/vaadin-crud/src/vaadin-crud.js

```diff
@@ -616,6 +616,11 @@ class CrudElement extends ElementMixin(ThemableMixin(PolymerElement)) {
     this.$.dialog.$.dialog.$.overlay.addEventListener('vaadin-overlay-escape-press', this.__cancelBound);
   }
 
+  /** @private */
+  __isSaveBtnDisabled(isDirty) {
```

Comment:
We should leave a comment here to indicate why this exists, that it's kind of an internal / non-public API for customizing this behaviour. From just looking at the web component only, future devs could be tempted to refactor / simplify this back to the old solution.

Reply:
Makes sense! Have added a comment.